backend/src/SessionRepository.js

```diff
@@ -0,0 +1,26 @@
+class SessionRepository {
```

Comment:
Please create an in-memory implementation of this repository too (we need both) because we will not run Hanghub on Heroku with Redis at this moment.

Reply:
Done.